src/index.js

```diff
@@ -154,6 +154,7 @@ class MultipleEntityRow extends LitElement {
     renderIcon(stateObj, config) {
         return html`<state-badge
             class="icon-small"
+            .hass=${this._hass}
```

Comment:
Not sure this gonna fix the problem, the `ha-state-icon` must be used here, doesn't it?
https://developers.home-assistant.io/blog/2024/01/30/ha-state-icon-properties-changes/

Reply:
The `ha-state-icon` component is already in use within `state-badge`.

I've just tested the proposed fix by overriding the code directly using Chrome dev tools, and it surely fixes the problem as it should.

Reply:
Very good!
Tried to add the change into obfuscated local js file, but still got those "bookmarks".
Could you explain how you managed to "override the code" by Code Inspector?

Reply:
Heh, just described that on the #330, please take a look there.

Reply:
Yes, managed to add a local change: https://github.com/benct/lovelace-multiple-entity-row/issues/330#issuecomment-1937355670